test/make.sh

```diff
@@ -67,14 +79,28 @@ function docker() {
     | compat_xargs -0 hadolint
 }
 
-# This function runs 'terraform validate' against all
-# directory paths which contain *.tf files.
+# This function runs 'terraform validate' and 'terraform fmt'
```

Comment:
`function docker` should be removed because of standard container usage, we don't need check dockerfiles

Reply:
same, for now we have to keep dockerfile in project